test/hotel_controller_test.rb

```diff
@@ -0,0 +1,113 @@
+require_relative "test_helper"
+
+describe Hotel::HotelController do
+  before do
+    @hotel_controller = Hotel::HotelController.new
+    @date = Date.parse("2020-08-04")
+  end
+  describe "hotelcontroller" do
+    describe "rooms" do
+      it "returns a list" do
+        rooms = @hotel_controller.rooms
+        expect(rooms).must_be_kind_of Array
+      end
+    end
+  end
+     describe "list_reservations_for_room" do
+      it "returns a list of reservations that fall with in the given date range for a specific room." do
```

Comment:
You should also have an edge case test for the situation where there are no reservations for a given room.